Tools/jmavsim_run.sh

```diff
@@ -10,7 +10,7 @@ extra_args=
 baudrate=921600
 device=
 ip="127.0.0.1"
-while getopts ":b:d:p:qsr:f:i:l" opt; do
+while getopts ":b:d:p:qsr:f:i:l:o" opt; do
```

Comment:
```suggestion
while getopts ":b:d:p:qsr:f:i:lo" opt; do
```

Reply:
I don't get it. Can you explain me why, please?

Reply:
A `:` marks that the flag takes an argument, and `-l` does not (whereas `-p <port>` for example does)